src/vrl/selectielijst/management/commands/load_data_from_excel.py

```diff
@@ -0,0 +1,137 @@
+        generiek_nummer = int(re.match(r"\d+\.(\d+)\.\d+", raw['Nr.']).group(1))
+        clean_data['generiek_resultaat'] = Resultaat.objects.filter(proces_type__id=proces_type.id,
+                                                                    nummer=generiek_nummer,
+                                                                    generiek_resultaat__isnull=True)[0]
+    else:
+        clean_data['generiek_resultaat'] = None
+
+    clean_data['nummer'] = int(re.match(r".*\.(\d+)", raw['Nr.']).group(1))
+    clean_data['naam'] = raw['Resultaat']
+    clean_data['omschrijving'] = raw['Omschrijving']
+    clean_data['herkomst'] = raw['Herkomst']
+    clean_data['waardering'] = check_choice(raw['Waardering'], ArchiefNominaties.labels)
+
+    if ',' in raw['Procestermijn'].replace('(', ','):
+        opmerking, procestermijn = raw['Procestermijn'].replace('(', ',').split(',')
+    else:
+        opmerking, procestermijn = '', raw['Procestermijn']
+    clean_data['procestermijn'] = check_choice(procestermijn, Procestermijnen.labels)
+    clean_data['procestermijn_opmerking'] = opmerking
+    clean_data['bewaartermijn'] = parse_duration(raw['Bewaartermijn'])
+    clean_data['toelichting'] = raw['Toelichting']
+    clean_data['algemeen_bestuur_en_inrichting_organisatie'] = bool(raw['Algemeen bestuur en inrichting organisatie'])
+    clean_data['bedrijfsvoering_en_personeel'] = bool(raw['Bedrijfsvoering en personeel'])
+    clean_data['publieke_informatie_en_registratie'] = bool(raw['Publieke informatie en registratie'])
+    clean_data['burgerzaken'] = bool(raw['Burgerzaken'])
+    clean_data['veiligheid'] = bool(raw['Veiligheid'])
+    clean_data['verkeer_en_vervoer'] = bool(raw['Verkeer en vervoer'])
+    clean_data['economie'] = bool(raw['Economie'])
+    clean_data['onderwijs'] = bool(raw['Onderwijs'])
+    clean_data['sport_cultuur_en_recreatie'] = bool(raw['Sport, cultuur en recreatie'])
+    clean_data['sociaal_domein'] = bool(raw['Sociaal domein'])
+    clean_data['volksgezonheid_en_milieu'] = bool(raw['Volksgezondheid en milieu'])
+    clean_data['vhrosv'] = bool(raw['VHROSV'])
+    clean_data['heffen_belastingen'] = bool(raw['Heffen belastingen etc'])
+    clean_data['alle_taakgebieden'] = bool(raw['Alle taakgebieden'])
+    return clean_data
+
+
+class Command(BaseCommand):
+    help = 'Load data from excel file to ProcesType and Resultaat'
+
+    def add_arguments(self, parser):
+        parser.add_argument('file_path', type=str, help='Path to excel file')
+
+    def handle(self, *args, **kwargs):
+        file_path = kwargs['file_path']
+        book = open_workbook(file_path)
+        sheet = book.sheet_by_index(0)
+
+        for raw in read_xls_row_to_dict(sheet):
+            # load to ProcesType
+            processtype_data = prepare_procestype(raw)
+            # if current nummer already exists - do nothing
+            p, created = ProcesType.objects.get_or_create(nummer=processtype_data['nummer'],
```

Comment:
I'd even expect an `update_or_create` - if minor text changes in 'Toelichting' happen for example, I'd want to update the database